2014/02/04 22:34:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/04 22:34:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/04 22:34:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/04 22:39:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpinfo".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('phpinfo')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/phpinfo
---
2014/02/04 22:39:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpinfo/info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('phpinfo/info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/phpinfo/info
---
2014/02/04 22:48:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpinfo".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('phpinfo')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/phpinfo/
---
2014/02/04 22:48:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 22:49:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 22:49:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 22:49:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 22:50:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 22:50:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 22:52:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 22:55:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "test/info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('test/info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/test/info
---
2014/02/04 22:56:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/04 23:00:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "nfo".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('nfo')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/nfo
---
2014/02/04 23:00:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://yii/site/login
---
2014/02/04 23:00:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('contact')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
HTTP_REFERER=http://yii/site/login
---
2014/02/04 23:00:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://yii/site/contact
---
2014/02/04 23:02:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:03:41 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "Info".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('Info', 'asd', true)
#1 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('Info', 'asd')
#2 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#7 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#8 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#10 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:05:02 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "Info".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('Info', Array, true)
#1 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('Info', Array)
#2 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#7 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#8 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#10 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:05:05 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "Info".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('Info', Array, true)
#1 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('Info', Array)
#2 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#7 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#8 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#10 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:09:17 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "contacts".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('contacts', 'asd', true)
#1 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('contacts', 'asd')
#2 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#7 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#8 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#10 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:09:18 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "contacts".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('contacts', 'asd', true)
#1 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('contacts', 'asd')
#2 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#7 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#8 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#10 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:12:00 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.Sitecontroller" is not defined.' in D:\xampp\htdocs\yii\www\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii\www\protected\views\site\index.php(22): CComponent->__get('Sitecontroller')
#1 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', 'asd', true)
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(869): CBaseController->renderFile('D:\xampp\htdocs...', 'asd', true)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('index', 'asd', true)
#5 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('index', 'asd')
#6 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#7 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#11 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#12 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#14 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:12:29 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.info" is not defined.' in D:\xampp\htdocs\yii\www\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii\www\protected\views\site\index.php(22): CComponent->__get('info')
#1 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', 'asd', true)
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(869): CBaseController->renderFile('D:\xampp\htdocs...', 'asd', true)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('index', 'asd', true)
#5 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('index', 'asd')
#6 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#7 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#11 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#12 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#14 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:21:30 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.info" is not defined.' in D:\xampp\htdocs\yii\www\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii\www\protected\views\site\index.php(22): CComponent->__get('info')
#1 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', NULL, true)
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(869): CBaseController->renderFile('D:\xampp\htdocs...', NULL, true)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#5 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('index')
#6 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#7 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#11 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#12 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#14 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:22:20 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.info" is not defined.' in D:\xampp\htdocs\yii\www\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii\www\protected\views\site\index.php(22): CComponent->__get('info')
#1 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', NULL, true)
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(869): CBaseController->renderFile('D:\xampp\htdocs...', NULL, true)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#5 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('index')
#6 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#7 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#11 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#12 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#14 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:22:54 [error] [php] Object of class SiteController could not be converted to string (D:\xampp\htdocs\yii\www\protected\views\site\index.php:22)
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(869): SiteController->renderFile()
#1 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): SiteController->renderPartial()
#2 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): SiteController->render()
#3 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): SiteController->runAction()
#6 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): SiteController->run()
#8 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 D:\xampp\htdocs\yii\www\index-test.php(15): CWebApplication->run()
REQUEST_URI=/site/info
2014/02/04 23:23:44 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.info" is not defined.' in D:\xampp\htdocs\yii\www\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii\www\protected\views\site\index.php(22): CComponent->__get('info')
#1 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\yii\www\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', 'asd', true)
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(869): CBaseController->renderFile('D:\xampp\htdocs...', 'asd', true)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('index', 'asd', true)
#5 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('index', 'asd')
#6 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#7 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#11 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#12 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#14 {main}
REQUEST_URI=/site/info
---
2014/02/04 23:25:16 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "info".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('info', Array, true)
#1 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(101): CController->render('info', Array)
#2 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionInfo()
#3 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('info')
#7 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/info')
#8 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#10 {main}
REQUEST_URI=/site/info
---
2014/02/05 15:08:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "controller/action".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('controller/acti...')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2014/02/05 15:08:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/05 15:08:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/05 15:08:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/05 15:08:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/05 15:09:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/05 15:09:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/05 15:09:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/05 15:09:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/05 15:09:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/05 15:09:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "info".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('info')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/info
---
2014/02/05 22:39:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/05 22:39:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/05 22:39:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/02/05 22:39:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in D:\xampp\htdocs\yii\www\framework\db\CDbConnection.php:382
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\xampp\htdocs\yii\www\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\xampp\htdocs\yii\www\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\xampp\htdocs\yii\www\framework\base\CModule.php(103): CModule->getComponent('db')
#4 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(105): CModule->__get('db')
#5 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(49): SiteController->actionVotings()
#6 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('votings')
#10 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/votings')
#11 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#13 {main}
REQUEST_URI=/votings
HTTP_REFERER=http://yii/
---
2014/02/05 23:31:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "vote".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(270): CController->missingAction('vote')
#1 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('vote')
#2 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/vote')
#3 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vote?voting=1
HTTP_REFERER=http://yii/votings
---
2014/02/05 23:31:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "vote".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(270): CController->missingAction('vote')
#1 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('vote')
#2 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/vote')
#3 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vote?voting=1
HTTP_REFERER=http://yii/votings
---
2014/02/05 23:36:43 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "voting".' in D:\xampp\htdocs\yii\www\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\www\framework\web\CController.php(782): CController->renderPartial('voting', Array, true)
#1 D:\xampp\htdocs\yii\www\protected\controllers\SiteController.php(117): CController->render('voting', Array)
#2 [internal function]: SiteController->actionVoting('1', NULL)
#3 D:\xampp\htdocs\yii\www\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#4 D:\xampp\htdocs\yii\www\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#5 D:\xampp\htdocs\yii\www\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 D:\xampp\htdocs\yii\www\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 D:\xampp\htdocs\yii\www\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(282): CController->run('voting')
#9 D:\xampp\htdocs\yii\www\framework\web\CWebApplication.php(141): CWebApplication->runController('site/voting')
#10 D:\xampp\htdocs\yii\www\framework\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\xampp\htdocs\yii\www\index-test.php(15): CApplication->run()
#12 {main}
REQUEST_URI=/voting?voting=1&name=%D0%9A%D0%B0%D0%BA%D1%83%D1%8E%20%D1%86%D0%BC%D1%81%20%D0%B2%D1%8B%D0%B1%D0%B5%D1%80%D0%B5%D1%82%D0%B5?
HTTP_REFERER=http://yii/votings
---
